customfieldnames.civix.php

```diff
@@ -346,3 +346,166 @@ function _customfieldnames_civix_civicrm_alterSettingsFolders(&$metaDataFolders
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @param string $formName
+ * @param CRM_Core_Form $form
+ */
+function _customfieldnames_civix_civicrm_buildForm($formName, &$form) {
+  $templatePath = realpath(dirname(__FILE__)."/templates");
+  if( ($formName == 'CRM_Custom_Form_Group') && ($form->getAction() == CRM_Core_Action::ADD) ) {
+    $templatePath = realpath(dirname(__FILE__)."/templates");
+    $form->add('text', 'table_name', ts('Set Table Name'), '', FALSE);
```

Comment:
let's just label it "Table Name"

Reply:
when CiviCRM creates a custom table via the UI in the normal core behavior, the table names are all prefixed with: civicrm_value_TABLENAME. they don't need to be constructed that way, and one benefit of this extension is to allow more flexibility. however, I'd like to set "civicrm_table_" as the default value for the table name field to "encourage" people to retain the standard naming convention.

Reply:
Set the label to "Table name". Regarding "civicrm_table_", as per my understanding I have set the default value for table_name field. Please check and confirm if it is the desired solution or something else.